styles.css

```diff
@@ -0,0 +1,99 @@
+.my-name {
+  font-family: 'Playfair Display', serif;
+  font-family: 'Poppins', sans-serif;
+  width: 220px;
+  height: 100px;
+  font-size: 50px;
+  margin-left: 16px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.contain {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  width: 320px;
+  padding-left: 0;
```

Comment:
- [ ] Please remove this with style so your hamburger menu will be not be fixed but feels responsive

Your Design
![image](https://user-images.githubusercontent.com/50941074/158902018-acd10510-4c78-4d6e-a9d3-c7b662638ca7.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/158902085-3b7bb0aa-6ad2-4f86-a99a-1a1f6e79699e.png)

Please fix this